crates/host/src/wasmbus/config.rs

```diff
@@ -1,7 +1,6 @@
 use crate::OciConfig;
 
-use std::sync::Arc;
-use std::time::Duration;
+use std::{collections::HashMap, sync::Arc, time::Duration};
```

Comment:
nit; can we keep the existing organization here? At the very least it will minimize diffs in the future and simplify resolving merge conflicts.

Reply:
Sure, and I agree with Brooks: if we care about these sorts of things, we should have them be automatically formatted